src/Sentry.Unity.Editor/ConfigurationWindow/OptionsConfigurationTab.cs

```diff
@@ -71,21 +69,18 @@ internal static class OptionsConfigurationItem
             return result;
         }
 
-        private static string SentryAssetPath(string scriptName, bool isEditorScript) =>
-            string.Format("Assets/{0}/Sentry/{1}.asset", isEditorScript ? "Plugins" : "Resources", scriptName);
+        private static string SentryAssetPath(string scriptName) => $"Assets/Resources/Sentry/{scriptName}.asset";
 
         private static void CreateScript<T>(string fieldName, string scriptName)
         {
-            var isEditorScript = typeof(T) == typeof(BuildtimeConfiguration);
-            var directory = isEditorScript ? "Assets/Editor" : "Assets/Scripts";
-
+            const string directory = "Assets/Scripts";
```

Comment:
Have you tested this with the integration test? The build-time script had buiold-time Unity (not Sentry) dependencies, so it had to reside in `Assets/Editor`

Reply:
Yes. That's why I moved the `CliOptions` to `Sentry.Unity` to get rid of the `Sentry.Unity.Editor` dependency in the options. Effectively, this means that the Buildtime scriptable object now lives inside of `Resources` and is part of the build but we got rid of all the blind casting and weird namespacing/dependencies.